background/get.js

```diff
@@ -129,3 +129,8 @@ async function getTabsAccessedOnDay(offset) {
     };
     return (await queryTabs()).filter(isLastAccessedAtDate);
 }
+
+/* Clear selection */
+export async function clearSelection(targetTab) {
+  return [ targetTab ];
```

Comment:
This is not the expectation if you invoke it via context menu. This command should return the active tab.